content/recipes/introduction/FAIR-and-knowledge-graphs.md

```diff
@@ -431,67 +396,7 @@ In the previous 2 sections, we have highlighted 2 of the most successful project
 Property Graphs as data structure for storing information and building knowledge graphs. 
 However, none of these implementations are officially an approved W3C standards. While it does not really matter
 at this stage, it was realized that such fragmentation was hurting and in 2019, the `open Graph Query Language` (oGQL) 
```

Comment:
Can we redirect the users to the oGQL specifications?

Reply:
the catch with this is that it seems the project stalled a bit after the initial 2019 impetus.